dist/images/run-smart-nic-ovnkube-node.sh

```diff
@@ -10,5 +10,4 @@ docker run --pid host --network host --user=0 --name ovn-node -dit --cap-add=NET
   -e OVN_KUBERNETES_NAMESPACE=ovn-kubernetes -e OVN_SSL_ENABLE=no -e OVNKUBE_NODE_MODE="smart-nic" \
   -e OVN_ENCAP_IP=$SMART_NIC_IP -e K8S_TOKEN=$K8S_TOKEN -e K8S_CACERT=$K8S_CACERT \
   -e OVN_GATEWAY_OPTS="$OVN_GATEWAY_OPTS" -e OVNKUBE_NODE_MGMT_PORT_NETDEV="$OVNKUBE_NODE_MGMT_PORT_NETDEV" \
-  -e OVN_DISABLE_PKT_MTU_CHECK=true \
```

Comment:
Should this be part of the previous "Handling packet sizes greater than pod MTU" patch?

Reply:
Or did I miss another commit we should revert?

Reply:
This file was added in one of the SmartNIC commits after the OVN_DISABLE_PKT_MTU_CHECK was introduced.
I thought it would be simple to remove this line manually than reverting the SmartNIC one (which results in large conflicts since there is a series of SmartNIC commits).

Reply:
Ok thanks for the explanation. I agree.